riff-config/src/main/java/com/wepay/riff/config/AbstractConfig.java

```diff
@@ -171,3 +176,4 @@ public Parser withDefault(Object defaultValue) {
     }
 
 }
+
```

Comment:
nit: In IntelliJ, it's showing me 2 new lines at the end of the file with this fix. Check it out once and fix it if its duplicate.

Reply:
thanks, fixed!